assets/scripts/index.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+const setAPIOrigin = require('../../lib/set-api-origin');
+const config = require('./config');
+
+$(() => {
+  setAPIOrigin(location, config);
```

Comment:
My heuristic is that "words" other than the first start with a capital letter and that I would never write `Api` (it looks odd to me), so `setAPIOrigin` is the only option left.